finals-2024-01-15/code/5265-genci-telqiu/src/app/keyvalue.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'keyvalue',
+})
+export class KeyValuePipe implements PipeTransform {
```

Comment:
The `keyvalue` pipe is already built-in into angular, so no need to reimplement it. Moreover, it's not even used in the app (it could be used with the oscars). However, +5 extra points for the effort